habitat/core/simulator.py

```diff
@@ -229,6 +154,23 @@ def get_observation(self, *args: Any, **kwargs: Any):
         raise NotImplementedError
 
 
+class BumpSensor(Sensor):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Yes, we need to make sure that all `Sensor`'s children methods have `*args: Any, **kwargs: Any` in the end of argument lists.